src/client/scss/app.scss

```diff
@@ -31,6 +31,42 @@
     cursor: ns-resize;
 }
 
+.zoom-buttons-wrapper {
+    position: absolute;
+    top: 74px;
+    right: 30px;
+    display: flex;
+}
+
+.zoom-buttons-wrapper button {
```

Comment:
In this file, Bootstrap styles are used to avoid specifying colors, sizes etc. explicitly, to maintain a consistent style across the whole app. Please use Bootstrap styles as much as possible.

Reply:
Changed it in [this commit](https://github.com/tilk/digitaljs_online/pull/34/commits/624659632023675806103264a07c45a1526378c2).